src/Analysis/GWASResults/Views/Home/Home.jsx

```diff
@@ -32,4 +34,9 @@ const Home = () => {
     </React.Fragment>
   );
 };
+
+Home.propTypes = {
+  selectedTeamProject: PropTypes.string.isRequired,
```

Comment:
I think storybook and test will need to be updated to contain this as well.

Reply:
Good catch, updated and confirmed unit tests and storybook are working now.